HW04.Rmd

```diff
@@ -18,9 +20,26 @@ output: github_document
 The gun_deaths.csv is provided in the repository. 
 ```{r load-data, message=FALSE, warning=FALSE}
 library(tidyverse)    # load tidyverse packages, including ggplot2
+library(knitr)
+library(ggrepel)
 
 # read in the data
+gun_deaths <- read.csv("gun_deaths.csv")
 
+# check variables in the data
+glimpse(gun_deaths)
+
+# convert age from character to numeric 
+gun_deaths$age <- as.numeric(gun_deaths$age)
+
+# clean up the labels of race
+gun_deaths$race <- gsub(pattern = "Not Availabletive American/Not Availabletive Alaskan", 

```

Comment:
It is nice to clean up the dataset a bit before starting the anlysis

Reply:
I agree!